autopts/wid/pbp.py

```diff
@@ -229,13 +233,11 @@ def hdl_wid_552(params: WIDParams):
     btp.cap_broadcast_source_setup_subgroup(source_id, subgroup_id, coding_format, vid, cid,
                                             codec_ltvs_bytes, metadata)
 
-    # Zephyr stack generates Broadcast_ID itself
```

Comment:
If there is not IXIT value for broadcast ID for PBP (...) then I suggest that we replace this comment with stating that, and that any broadcast_id will work.